client/public/apiary-proxy.js

```diff
@@ -21,7 +23,37 @@ Apiary.buildChannel = function (origin, scope) {
         errorCopy.message = err.message;
         trans.error(errorCopy);
       })
-  })
-}
+  });
+
+  return chan;
+};
+
+Apiary.createAgent = function createAgent({ subdomains }) {
+  if (!Array.isArray(subdomains)) {
+    throw new Error('subdomains property must be an array');
```

Comment:
I think it's more of an `option` than a `property` - at least for the end user, even tho `property` is a correct term.